js/views/session_change_nickname_dialog_view.js

```diff
@@ -0,0 +1,71 @@
+/* global Whisper */
+
+// eslint-disable-next-line func-names
+(function() {
+  'use strict';
+
+  window.Whisper = window.Whisper || {};
+
+  Whisper.SessionNicknameDialog = Whisper.View.extend({
+    className: 'loki-dialog session-nickname-wrapper modal',
+    initialize(options) {
+      this.props = {
+        title: options.title,
+        message: options.message,
+        messageSub: options.messageSub,
+        onClickOk: this.ok.bind(this),
+        onClickClose: this.cancel.bind(this),
+        resolve: options.resolve,
+        reject: options.reject,
+        okText: options.okText,
+        cancelText: options.cancelText,
+        okTheme: options.okTheme,
+        closeTheme: options.closeTheme,
+        hideCancel: options.hideCancel,
+        sessionIcon: options.sessionIcon,
+        iconSize: options.iconSize,
+        convoId: options.convoId,
+        placeholder: options.placeholder,
```

Comment:
no need for all those args. You can just keep the one you need you actually need to be dynamic.